verifier/verifier.py

```diff
@@ -43,30 +43,42 @@ def __init__(self, code, response):
         self.code = code
         self.response = response
 
+
+####
+# SMTP RCPT error handlers
+####
 def handle_550(response):
     if any([keyword.encode() in response for keyword in blocked_keywords]):
-        return "Blocked by mail server"
-    return "Mailbox unavailable"
-
-def handle_551(response):
-    return "Mailbox has moved"
-
-def handle_421(response):
-    return "Try again later"
+        message = "Blocked by mail server"
+    else:
+        message = "Mailbox unavailable"
```

Comment:
In this case, there should be no message, right? Because this is sure that mailbox does not exist thus making it not deliverable. Same is for 450.  (though if blocked there should be a message) Also, please squash all related commit into one. 
Adding .idea and removing space - 1 commit :+1: 
Adding exception handling - 1 commit.

:+1: